tests/unit/test_constants.py

```diff
@@ -0,0 +1,11 @@
+from openscm import constants
+from datetime import timedelta
+
+
+def test_constants():
+    assert constants.MINUTE == timedelta(minutes=1).total_seconds()
+    assert constants.HOUR == timedelta(hours=1).total_seconds()
+    assert constants.DAY == timedelta(days=1).total_seconds()
+    assert constants.WEEK == timedelta(weeks=1).total_seconds()
+    assert constants.MONTH == timedelta(days=30).total_seconds()
```

Comment:
```suggestion
    
```

Surely we don't want to specify that a month is always 30 days yet? (If we later need to, we do it then)